### ISSUE TITLE
📜Single or double quotation marks

### ISSUE THREAD

**Issue by MarleenGilsing** [hedyorg/hedy#5899]
While translating Feliennes worksbooks I saw a lot of exercises that contain either single or double quotation marks. We don't use double quotes in the adventures, only singles, but I think it makes a lot of sense to use the double ones, or at least teach the students that you can use doubles to prevent problems with contractions. Should we add this to the adventures?

**Comment by Felienne** [comment]
Hi @MarleenGilsing!

I principle I prefer single quote because they are easier to type, so I would suggest doing that everywhere where it is possible.

And we do explain about contractions or hoge komma's in Dutch:

![Image](https://github.com/user-attachments/assets/7de072b2-5472-49dc-8f27-7ee54afe9845)

![Image](https://github.com/user-attachments/assets/373dfa67-9214-457d-a919-95d1655a87a2)

**Comment by MarleenGilsing** [comment]
Yes we explain the problem but our solution is to use incorrect spelling. Shouldn't the solution be to use double quotes instead?

**Comment by Felienne** [comment]
> Yes we explain the problem but our solution is to use incorrect spelling. Shouldn't the solution be to use double quotes instead?

Ah yes, I see now!! I have never taught it to my students this way haha, I tell them to use double quotes (see also the worksheets)

I am only now realizing our lessons do not teach that, and you are right, we should!

**Comment by MarleenGilsing** [comment]
Yeah I only just found out you're allowed to do that haha. 🙈
But I'll change it so there is no more ugly spelling!
